cypress/integration/edit.js

```diff
@@ -88,9 +88,6 @@ describe('Edit Page', () => {
             .clear()
             .type('Sed quo');
         cy.get('[role="tooltip"]').within(() => {
-            cy.contains(
```

Comment:
this is a regression

Reply:
Yes, we talked about it and you said ok for it

Reply:
Gotcha